### DISCUSSION TITLE
Copilot Staus showing conflicting messages

### DISCUSSION THREAD

**Discussion by erikiva** [community/community#30233] [Copilot Conversations]
When I go to my settings I am seeing both that I am elegible for a free account and that it is going to expire in two days, any idea which is it? 

> Your free access to GitHub Copilot is ending on August 22, 2022 at 9am PDT.
> You are eligible to use GitHub Copilot for free.

![Screen Shot 2022-08-20 at 4 41 34 PM](https://user-images.githubusercontent.com/13215331/185765296-27410c33-7d80-486c-a2e2-b50013b4d9b7.png)

**Comment by radar-blip** [comment]
Depends on if you Student and/or Maintainer of open source projects.

Here is some more info: 
https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot

**Reply by radar-blip** [reply]
Looks like you are eligible, so why not?

**Reply by erikiva** [reply]
I just don't understand if I am or not because on the one hand it says I am and on the other it says that my free access is ending. I have no idea what github considers a popular open source project 🤷‍♀️

**Reply by radar-blip** [reply]
Maybe what they really mean is, you are eligible for a free trial. 
Not that they would try to trick you into signing up for something. 😄
